src/context/App.tsx

```diff
@@ -166,11 +165,8 @@ export const AppProvider = ({ children }: { children: ReactNode }) => {
 
   const validateToken = useCallback(
     async ({ token, hostname }: AuthTokenOptions) => {
-      await apiRequestAuth(
-        `${generateGitHubAPIUrl(hostname)}notifications`,
-        'HEAD',
-        token,
-      );
+      await headNotifications(hostname, token);
```

Comment:
This name is weird, but correct I suppose 😅

Reply:
Super weird.  I was trying to think _why_ we need this specific call at all...

Reply:
Considering that this lives under the validateToken, I assume that it does just that 😅 Testing the token with a simple and light request just to test the permissions I suppose. BUT, we could also do it the other way, and test the token directly by looking at the scopes, similar to how we were doing in the color scopes (?)

Reply:
Future us problem haha